core/lib/config/src/configs/experimental.rs

```diff
@@ -60,3 +61,9 @@ impl ExperimentalDBConfig {
         100
     }
 }
+
+#[derive(Debug, Clone, Default, PartialEq, Deserialize)]
+pub struct ExperimentalVmConfig {
+    /// Operation mode for the new fast VM. `None` to disable the fast VM.
+    pub fast_vm_mode: Option<FastVmMode>,
```

Comment:
Not sure whether `Option<_>` makes more sense compared to adding an additional variant to `FastVmMode`. WDYT?

Reply:
I think a dedicated variant would be more expressive, though either way is fine with me.

Reply:
I prefer old, new, shadow over Option(new, shadow)